easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru rmats-turbo-4.1.1.orig/rMATS_C/Makefile rmats-turbo-4.1.1/rMATS_C/Makefile
```

Comment:
We really like to have a small description of what the patch does (and who wrote it)